appd.adoc

```diff
@@ -317,7 +313,7 @@ f(j,i)= 0.5*(z1+ z2) + 0.5*(z1-z2)* tanh(2*a/(z1-z2)*(depth(j,i)-href))
               
 ----
 
-where **`z(k,j,i)`**is height, positive upwards, relative to ocean datum (e.g. mean sea level) at gridpoint **`(k,j,i)`**, **`sigma(k)`**is the dimensionless coordinate at vertical gridpoint **`(k)`**for **`k &lt;= k_c`**, and **`depth(j,i)`**is the distance from ocean datum to sea floor (positive value) at horizontal gridpoint **`(j,i)`**. z1, z2, a, and hrefare constants.
+where **`z(k,j,i)`**is height, positive upwards, relative to ocean datum (e.g. mean sea level) at gridpoint **`(k,j,i)`**, **`sigma(k)`**is the dimensionless coordinate at vertical gridpoint **`(k)`**for **`k &lt;= k_c`**, and **`depth(j,i)`**is the distance from ocean datum to sea floor (positive value) at horizontal gridpoint **`(j,i)`**. **`z1`**, **`z2`**, **`a`**, and **`href`**are constants.
```

Comment:
Would you mind checking the other inline-code fragments in this paragraph and adding trailing spaces as appropriate?